dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -239,10 +287,40 @@ def install(self):
             print("Build DALI TF library:\n\n " + cmd + '\n\n')
             subprocess.check_call(cmd, cwd=self.src_path, shell=True)
 
-            if not self.check_import(lib_path, dali_stub_lib):
-                raise ImportError("Error while importing the DALI TF plugin built from source, will not install")
+            if not self.check_plugin(lib_path, dali_stub_lib):
+                raise ImportError("Error while loading or testing the DALI TF plugin built from source, will not install")
             print("Installation successful")
 
+    def check_install_env(self):
+        print("Checking build environment for DALI TF plugin ...")
+        print(self.debug_str())
+
+        Path(self.plugin_dest_dir).mkdir(parents=True, exist_ok=True)
+
+        if not self.tf_version or not self.tf_path:
+            error_msg = "Installation error:"
+            error_msg += "\n Tensorflow installation not found. Install `tensorflow-gpu` and try again"
+            error_msg += '\n' + self.debug_str()
+            raise ImportError(error_msg)
+
+    def install(self):
+        self.check_install_env()
+
+        if self.prebuilt_exact_ver:
+            if self.install_prebuilt():
+                return
```

Comment:
Probably very nitpicky

```suggestion
        if self.prebuilt_exact_ver and self.install_prebuilt():
           return
```